src/main/java/org/springframework/data/elasticsearch/repository/query/ElasticsearchQueryMethod.java

```diff
@@ -123,6 +125,23 @@ private HighlightQuery createAnnotatedHighlightQuery() {
 				getDomainClass());
 	}
 
+	/**
+	 * @since 4.4
+	 * @return if the method has additional configuration for querying
+	 */
+	public boolean hasValueParams() {
+		return hasAnnotatedQuery() && !queryAnnotation.valueParams().equals("");
+	}
+
+	/**
+	 * @since 4.4
+	 * @return the additional configuration for querying
+	 * @throws JsonProcessingException if the json or elasticsearch argument is malformed
+	 */
+	public String getValueParams() throws JsonProcessingException {
```

Comment:
where would the exception been thrown?

Reply:
sorry will remove this, was thrown in previous implementation